<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Input</class>
 <widget class="QWidget" name="Input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#2f3542;
border-radius: 1px;
font: 9pt &quot;Arial&quot;;
font-size: 20px;
border: 2px solid rgb(85, 0, 255);</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>581</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 6px;
font: 9pt &quot;Arial&quot;;
font-size: 20px;
border: 2px solid rgb(85, 0, 255);
background-color: #2d3436;
color:#00cec9</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:20px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InputCode">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 6px;
font: 9pt &quot;Arial&quot;;
font-size: 20px;
border: 2px solid rgb(85, 0, 255);
background-color: #000000;
color: #00FF00;</string>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 6px;
font: 9pt &quot;Arial&quot;;
font-size: 20px;
border: 2px solid rgb(85, 0, 255);
background-color: #000000;
color: #00FF00;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
border-radius: 14px;
font: &quot;Arial&quot;;
font-size: 16px;
border: 2px solid rgb(98, 98, 98);
background-color: #000000;</string>
   </property>
   <property name="text">
    <string>â– </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
